plugin-server/src/utils/db/db.ts

```diff
@@ -1220,10 +1220,19 @@ export class DB {
 
     // SessionRecordingEvent
 
-    public async fetchSessionRecordingEvents(): Promise<PostgresSessionRecordingEvent[] | SessionRecordingEvent[]> {
+    /* Either gets all session recording events since the database was last reset
+     * or, if a sessionId is provided, gets all session recording events for that sessionId
+     * @param sessionId
+     */
+    public async fetchSessionRecordingEvents(
+        sessionId?: string | undefined
+    ): Promise<PostgresSessionRecordingEvent[] | SessionRecordingEvent[]> {
+        const predicate = !!sessionId ? ` WHERE session_id = '${sessionId}'` : ''
```

Comment:
Are we sure inlining the ID in the query without any sanitization is a good idea?